public/html/ticTacToe.html

```diff
@@ -0,0 +1,48 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Tic Tac Toe</title>
+
+
+        <style>
+
+            body {
+                width: 550px;
+                mrgin: auto;
+            }
+
+            h1 {
+                border-bottom: 1px solid black;
+            }
+
+            .Square {
+                height: 100px;
+                width: 100px;
+                font-size: 50px;
+                text-align: center;
+                font-weight:
+            }
+        </style>
+    </head>
+    <body>
+        <h1>Tic Tac Toe</h1>
```

Comment:
Please use two spaces instead of four, and please don't use tabs. https://google.github.io/styleguide/htmlcssguide.html#Indentation